.github/ISSUE_TEMPLATE/create-a-test.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Create a Test
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature you want to create a test**
+A clear and concise description of what the test is.
+Ex.
+```
+As a [role]
+I want [feature]
+So that [benefit]
+```
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
suggestions: add list, with real user flow, so that folks can use it as a convention when they add a test request

```
- on page: <archive.org>
- with browser width: (desktop, tablet, mobile)
- i click on: <this button>
- I expect: <this to happen>

```